src/idea.js

```diff
@@ -1,18 +1,34 @@
 class Idea {
-  constructor() {
```

Comment:
I want to talk about the constructor for the constructor on line 2.  I know having two is discouraged for ES6.  Still, lets discuss in todays project meet.